Workflows/Architecture review.md

```diff
@@ -26,9 +26,30 @@ Please follow this timeline:
 8. Once you've finished the database design, it has to be approved by someone from the management.
 9. Once your architecture has been approved, you can generate a new Rails project. Read more about that on the next page.
 
+Ask the project manager to give you the project specification.
+
+## Database design & discuss major implementation details
+Once you're done with the initial project meeting and have the specification, you can start designing the database. Use an online tool so you can easily present the diagram later on the review, such as [dbdiagram](https://dbdiagram.io/home) or [SqlDBM](https://sqldbm.com/Home/), or some other.
```

Comment:
We mention tooling above as well, we should probably only do it once

Reply:
Also mention that the login credentials for sqldbm can be found on 1pass